circle.yml

```diff
@@ -0,0 +1,35 @@
+machine:
+  environment:
+    YARN_VERSION: 0.20.3
+    PATH: "${PATH}:${HOME}/.yarn/bin:${HOME}/${CIRCLE_PROJECT_REPONAME}/node_modules/.bin"
+    NODE_ENV: production
+  node:
+    version: 6.6.0
+
+dependencies:
+  pre:
+      - |
+        if [[ ! -e ~/.yarn/bin/yarn || $(yarn --version) != "${YARN_VERSION}" ]]; then
+          echo "Download and install Yarn."
+          curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version $YARN_VERSION
+        else
+          echo "The correct version of Yarn is already installed."
+        fi
+  override:
+      - yarn
+      - webpack --config webpack.config.js
+      - npm run update
+      - npm run build-server
```

Comment:
Yarn is used to install dependencies and npm is just piggybacking on package.json commands.

Yarn is much faster to install dependencies, wouldn't you agree?